forge/src/test/java/org/openjdk/skara/forge/github/GitHubRestApiTests.java

```diff
@@ -148,4 +148,15 @@ void testOversizeComment() {
         assertTrue(updateComment.body().contains("..."));
         assertTrue(updateComment.body().contains("2"));
     }
+
+    @Test
+    void testLatestBody(){
+        var testRepoOpt = githubHost.repository("openjdk/playground");
+        assumeTrue(testRepoOpt.isPresent());
+        var testRepo = testRepoOpt.get();
+        var testPr = testRepo.pullRequest("99");
+
+        String latestBody = testPr.latestBody();
```

Comment:
This looks a bit fragile, but maybe that's a price we have to pay for the manual tests.

However, I think you should also add an automatic test, that verifies that an updated body makes the bot skip the update that time around, but not otherwise. I think it will be possible to define such a setup without having to resort to race-prone logic.